src/packages/shared-types/authority.ts

```diff
@@ -6,9 +6,24 @@ export enum Authority {
   "1915c" = "1915(c)",
 }
 
-/* Authority values as a union string type. Replaces old Authority enum. */
 export type AuthorityUnion =
   | "Medicaid SPA"
   | "CHIP SPA"
   | "1915(b)"
   | "1915(c)";
+
```

Comment:
The constants are there to prevent us from sending typos (`mediciad spa` vs `medicaid spa`) when sending data to the backend. 
If I'm understanding you correctly, should I make a constant in the backend that we can import into the frontend and use the appropriate values?

Reply:
Is the constant name the issue? I can change to `SEATOOL_TO_FE` and vice versa. 
There's no major difference between my changes and the `Authority` enum. I was under the impression we're looking to move away from `enum`s (based on teammate conversations and the comment in the code on line 1), so I implemented constants with the union types.

Reply:
> And my recommendation if asked, would be to keep the enum, and move forward with the routing changes etc, as that's how I would do it if working the ticket

Yes, otherwise, all those unrelated changes will add noise to the PRs. I'll be on the lookout in PR reviews whenever they use the `Authority` enum to use the type union. What name would you prefer the constant to have? I'd like to convert values coming from the API to be user-friendly and vice versa.

Reply:
Ok, gonna replace it then

Reply:
Resolving with https://github.com/Enterprise-CMCS/macpro-mako/pull/644/commits/2477a612e1293966109adde5f6cd6f8814ed046e